pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+build-backend = 'poetry_dynamic_versioning.backend'
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+
+
+[tool.poetry]
+name = "reqstool-python-poetry-plugin"
+version = "0.0.1"
```

Comment:
We use dynamic versioning is this needed?

Reply:
according to the offical docs, it should be present, and set to 0.0.0 by default.